services/QuillLMS/client/app/bundles/Teacher/containers/UploadRosters.tsx

```diff
@@ -0,0 +1,83 @@
+import * as React from "react";
+import XLSX from 'xlsx'
+import request from 'request'
+
+import getAuthToken from '../components/modules/get_auth_token'
+
+
+export const UploadRosters = () => {
+
+  const [schoolId, setSchoolId] = React.useState<Number>();
+  const [teachers, setTeachers] = React.useState<Array<any>>([]);
+  const [students, setStudents] = React.useState<Array<any>>([])
```

Comment:
Love the functional component Cissy!

Reply:
Haha thank you I finally figured out how to do it XD